sycl/include/sycl/ext/intel/experimental/esimd.hpp

```diff
@@ -10,7 +10,41 @@
 
 #pragma once
 
-/// \defgroup sycl_esimd DPC++ Explicit SIMD API
+/// @defgroup sycl_esimd DPC++ Explicit SIMD API
```

Comment:
Nit. We use `\` in SYCL runtime, but I'm ok with `@` for ESIMD.

Reply:
Yes, for ESIMD I would like to use `@` as it is more eye catchy